glusterd2/xlator/global.go

```diff
@@ -37,6 +36,7 @@ func Xlators() map[string]*Xlator {
 
 // loadOptions loads all available options into the options.Options map,
 // indexed as <xlator-id>.<option-key> for all available option keys
+// Also initialise the globalOptMap with defauls/value from etcd server
```

Comment:
Is this comment belong to previous patchset?

Reply:
Yes it is! Will remove.